BLaurenB/techtime
# Freelancer Index and Show style: card structure

#### What's this PR do?
Adds card deck styling to the freelancer index and freelancer show.

#### Where should the reviewer start- what are the pertinent changes and the sequence?
freelancers/index.html.erb to see card styling in groups of 3 freelancers
freelancers/show.html.erb for centered card show

#### How should this be manually tested?
visit /freelancers to check if cards are 3 across.
visit /freelancers/1 to see if card is centered.

#### Any background context you want to provide?
#### What are the relevant tickets?
#### Screenshots (if appropriate)
#### Questions:
